R/Mexico.R

```diff
@@ -59,7 +59,7 @@ Mexico <- R6::R6Class("Mexico",
     },
 
     #' @description Data download function for Mexico data. This replaces the
-    #' generic download function in `DataClass`. To get the latest data
+    #' generic download function in `DataClass()`. To get the latest data
```

Comment:
`download()` or similar?

Reply:
Appologies, what do you mean by this?

Reply:
download is a method so should we be referring to it using `` and perhaps using `download()` to match with how we refer to other methods?

Reply:
I was just using it as a name for the download function. Can make it a reference but clicking on it wouldn't take you anywhere